SpinCore_pp/run_field_dep_justMw.py

```diff
@@ -88,162 +76,126 @@ def verifyParams():
-                            ('delay',deadtime),
-                            ('acquire',acq_time),
-                            ('delay',repetition),
-                            ('jumpto','start')
-                            ])
-                        #}}}
-                    print("\nSTOPPING PROG BOARD...\n")
-                    SpinCore_pp.stop_ppg();
-                    print("\nRUNNING BOARD...\n")
-                    SpinCore_pp.runBoard();
-                    raw_data = SpinCore_pp.getData(data_length, nPoints, nEchoes, nPhaseSteps, output_name)
-                    raw_data.astype(float)
-                    data_array = []
-                    data_array[::] = complex128(raw_data[0::2]+1j*raw_data[1::2])
-                    print("COMPLEX DATA ARRAY LENGTH:",shape(data_array)[0])
-                    print("RAW DATA ARRAY LENGTH:",shape(raw_data)[0])
-                    dataPoints = float(np.shape(data_array)[0])
-                    if x == 0 and B0_index == 0:
-                        time_axis = np.linspace(0.0,nEchoes*nPhaseSteps*acq_time*1e-3,dataPoints)
-                        data = ndshape([len(data_array),nScans,len(field_axis),1],['t','nScans','Field','power']).alloc(dtype=np.complex128)
-                        data.setaxis('t',time_axis).set_units('t','s')
-                        data.setaxis('nScans',r_[0:nScans])
-                        data.setaxis('Field',field_axis)
-                        data.setaxis('power',r_[powers])
-                        data.name(node_name)
-                        data.set_prop('acq_params',acq_params)
-                    data['nScans',x]['Field',B0_index]['power',0] = data_array
-                last_power = this_power
-            SpinCore_pp.stopBoard();
+total_pts = nPoints*nPhaseSteps
+assert total_pts < 2**14, "You are trying to acquire %d points (too many points) -- either change SW or acq time so nPoints x nPhaseSteps is less than 16384"%total_pts
+#{{{ppg
+def run_sweep(fieldaxis, nScans=1, B0_index, data = None):
+    print("About to run run_scans for", power_idx)
+    ph1_cyc = r_[0,1,2,3]
+    ph2_cyc = r_[0]
+    nPhaseSteps = len(ph1_cyc)*len(ph2_cyc)
+    total_pts = nPoints*nPhaseSteps
+    data_length = 2*nPoints*nEchoes*nPhaseSteps
+    for x in range(nScans):
+            print("\n*** *** *** *** ***\n")
+            print("\n*** *** ***\n")
+            print("CONFIGURING TRANSMITTER...")
+            SpinCore_pp.configureTX(adcOffset, carrierFreq_MHz, tx_phases, amplitude, nPoints)
+            print("\nTRANSMITTER CONFIGURED.")
+            print("***")
+            print("CONFIGURING RECEIVER...")
+            acq_time = SpinCore_pp.configureRX(SW_kHz, nPoints, nScans, nEchoes, nPhaseSteps) #ms
+            print("\nRECEIVER CONFIGURED.")
+            print("***")
+            print("\nINITIALIZING PROG BOARD...\n")
+            SpinCore_pp.init_ppg();
+            print("\nLOADING PULSE PROG...\n")
+            if phase_cycling:
```

Comment:
maybe delete this if -- see next question

Reply:
took out all things that have options for phase_cycling and set the phase_Cycling as default